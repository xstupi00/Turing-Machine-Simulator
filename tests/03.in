S a B b
B b C L
C   F a
a
