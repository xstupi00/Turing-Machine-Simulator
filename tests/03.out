ABNORMAL STOP: TS does not find the transitions to the final state F.
